frontend/integration-tests/tests/filter.scenario.ts

```diff
@@ -73,8 +73,6 @@ describe('Filtering', () => {
 
   it('filters Deployment from other namespace', async () => {
     await browser.get(`${appHost}/k8s/ns/kube-system/deployments`);
-    await browser.wait(until.elementToBeClickable(crudView.nameFilter), BROWSER_TIMEOUT);
-    await crudView.nameFilter.sendKeys(WORKLOAD_NAME);
```

Comment:
I'm confused why these lines are removed. Isn't this test specifically to check that filtering works?

Reply:
We are removing them because we don't show search filters in an empty page.